dom/schema/v1/jobimp.sql

```diff
@@ -0,0 +1,17 @@
+create table if not exists jobimp
+(
+    id           text not null primary key,
+    worker_id    text not null,
+    created      timestamptz,
+    updated      timestamptz,
```

Comment:
Should we use `timestamptz NOT NULL DEFAULT NOW()` for these?

Reply:
Agree. Will do when the job import format will be stabilized. I have only one sample file.